### ISSUE TITLE
In-line images not displaying correctly

### ISSUE THREAD

**Issue by rebloor** [mdn/content#10907]
## What page(s) did you find the problem on?

Various tools pages e.g.[Network Monitor](https://developer.mozilla.org/en-US/docs/Tools/Network_Monitor) where images of icons are used within sentences but by default displayed as block elements and therefore on separate lines.

<img width="812" alt="image" src="https://user-images.githubusercontent.com/7352080/144285427-45aa2529-230f-48ad-81c6-d12271813b8d.png">

These images need to be updated to include the `style="display: inline"` directive to display them in-line. They might also look better if resized e.g. `height="20" width="20"`

<img width="1120" alt="image" src="https://user-images.githubusercontent.com/7352080/144285942-255cbf6f-387f-445b-9664-3df21ecc84da.png">

**Comment by AnilSeervi** [comment]
Guess this is not the only page.
Here's another one - https://developer.mozilla.org/en-US/docs/Tools/Taking_screenshots
And there maybe more!

All the `<img>` tags on the page have a default styling of `display: block` applied here - [mdn/mdn-minimalist](https://github.com/mdn/mdn-minimalist/blob/main/sass/lib/_reset.scss#L54-L57)

**Comment by hamishwillee** [comment]
Yeah. So the fix is probably to use an img tag and apply the display-inline style. Probably you'd also size the image to the size you want (rather than specifying the height/width).
